requirements.txt

```diff
@@ -15,3 +15,4 @@ lmdb==0.87
 h5py>=2.2.1,<=2.6.0
 pydot==1.0.28
 psutil>=1.2.1,<=3.4.2
+matplotlib==1.5.1
```

Comment:
Oh sorry I am using a `virtualenv` and I usually get pretty up-to-date versions through `pip` so that is the version I added to the requirements. Due to some inextricable dependencies between packages I was unable to install matplotlib 1.3.1 in a virtual env (I promise I did try hard). I verified that your requirements directive is working (`matplotlib>=1.3.1,<=1.5.1`). Also, I checked that the API I am using in matplotlib (to deal with colour maps) was already in 1.3.1. I will update that line.